src/components/SearchBar/index.js

```diff
@@ -15,7 +15,7 @@ function SearchBar(props) {
 
   return (
     <div className="searchbar">
-      <FaSearch id="search-icon" onClick={handleSubmit} />
+      <FaSearch id="search-icon" onClick={handleSubmit}/>
```

Comment:
Actually, I don't think it's very obvious that clicking the icon is how you submit the search. I would suggest either of the following approaches:

Use onChange on the input itself so it searches as you type

Add a search button